tools/packaging/common/envoy_bootstrap_drain.json

```diff
@@ -3,7 +3,7 @@
     "access_log_path": "/dev/null",
     "address": {
       "socket_address": {
-        "address": "127.0.0.1",
+        "address": "{{ .localhost }}",
```

Comment:
is this configured somewhere?

Reply:
Yes, pilot-agent code has rendering logic, the same approach is used in `istio/tools/packaging/common/envoy_bootstrap_v2.json` 
This file was missed in initial ipv6 PR.